server/app/api/controllers/peer_review.js

```diff
@@ -21,11 +24,14 @@ module.exports = {
         },
         function (err, result) {
           if (err) next(err);
-          else
-            res.json({
+          else{
+             const peerName ="surekha.test.email";
+             emailProvider.sendEmail("surekha.gadkari@objectedge.com", 'Peer Review  ', emails.peerReviewEmailTemplate(peerName))
```

Comment:
Please use variable for hardcoded values.

Reply:
Done.